pyqtgraph/graphicsItems/PColorMeshItem.py

```diff
@@ -0,0 +1,253 @@
+from __future__ import division
+
+from ..Qt import QtGui, QtCore
+import numpy as np
+from .. import functions as fn
+from .. import debug as debug
+from .GraphicsObject import GraphicsObject
+from ..Point import Point
+from .. import getConfigOption
+from .GradientEditorItem import Gradients # List of colormaps
+from ..colormap import ColorMap
+
+try:
+    from collections.abc import Callable
+except ImportError:
+    # fallback for python < 3.3
+    from collections import Callable
+
+__all__ = ['PColorMeshItem']
+
+
+class PColorMeshItem(GraphicsObject):
+    """
+    **Bases:** :class:`GraphicsObject <pyqtgraph.GraphicsObject>`
+    """
+
+    sigImageChanged = QtCore.Signal()
+    sigRemoveRequested = QtCore.Signal(object)  # self; emitted when 'remove' is selected from context menu
+
+
+    def __init__(self, *args, **kwargs):
+        """
+        Create a pseudocolor plot with convex polygons.
+
+        Call signature:
+
+        PColorMeshItem([x, y,] z, **kwargs)
+
+        x and y can be used to specify the corners of the quadrilaterals.
+        z must be used to specified to color of the quadrilaterals.
+
+        Parameters
+        ----------
+        x, y : np.ndarray, optional, default None
+            2D array containing the coordinates of the polygons
+        z : np.ndarray
+            2D array containing the value which will be maped into the polygons
+            colors.
+            If x and y is None, the polygons will be displaced on a grid
+            otherwise x and y will be used as polygons vertices coordinates as:
+
+            (x[i+1, j], y[i+1, j])           (x[i+1, j+1], y[i+1, j+1])
```

Comment:
These diagrams probably need to be put in a code environment to render correctly.

Reply:
How can I do that?

Reply:
You can take a look at how matplotlib is doing it: https://matplotlib.org/3.3.0/_modules/matplotlib/axes/_axes.html#Axes.pcolormesh

Basically, you can make a double-colon `::` where you already have one and then indent the code block one level more than the text it's surrounded by.

Reply:
ok, it looks like this:

```python
"""
            otherwise x and y will be used as polygons vertices coordinates as::

                (x[i+1, j], y[i+1, j])           (x[i+1, j+1], y[i+1, j+1])
                                    +---------+
                                    | z[i, j] |
                                    +---------+
                    (x[i, j], y[i, j])           (x[i, j+1], y[i, j+1])
"""
```